drivers/sensor/mpu9250/mpu9250.h

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2021, Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_SENSOR_MPU9250_MPU9250_H_
+#define ZEPHYR_DRIVERS_SENSOR_MPU9250_MPU9250_H_
+
+#include <device.h>
+#include <drivers/gpio.h>
+#include <sys/util.h>
+#include <zephyr/types.h>
```

Comment:
please review include list here to make this header self-contained. For example, Kernel structs are being used but `kernel.h` is not directly included.

Reply:
Fixed